frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -376,10 +376,10 @@ export const PollIntervalDropdown: React.FC<{}> = () => {
   );
 };
 
-const TimeDropdowns: React.FC<{}> = React.memo(() => (
+const TimeDropdowns: React.FC<TimeDropdownsProps> = React.memo(({ namespace }) => (
```

Comment:
Since we already have `NamespaceContext` on this page, how about we use that instead of passing `namespace` around?